flowmachine/flowmachine/features/subscriber/subscriber_tacs.py

```diff
@@ -17,6 +17,62 @@
 from .metaclasses import SubscriberFeature
 from ...core import Table
 
+valid_characteristics = {
```

Comment:
Can we sort these alphabetically so that they are easier to parse at a glance?

For convenience, here is the sorted list:
```
valid_characteristics = {
    "brand",
    "depth",
    "display_colors",
    "display_height",
    "display_type",
    "display_width",
    "hardware_bluetooth",
    "hardware_edge",
    "hardware_gprs",
    "hardware_gps",
    "hardware_umts",
    "hardware_wifi",
    "height",
    "hnd_type",
    "j2me_cldc_10",
    "j2me_cldc_11",
    "j2me_cldc_20",
    "j2me_midp_10",
    "j2me_midp_20",
    "j2me_midp_21",
    "mms_built_in_camera",
    "mms_receiver",
    "model",
    "software_os_name",
    "software_os_vendor",
    "software_os_version",
    "syncml_dm_acc_gprs",
    "syncml_dm_app_bookmark",
    "syncml_dm_app_browser",
    "syncml_dm_app_internet",
    "syncml_dm_app_java",
    "syncml_dm_app_mms",
    "syncml_dm_settings",
    "tac",
    "wap_1_2_1",
    "wap_2_0",
    "wap_push_oma_app_browser",
    "wap_push_oma_app_ims",
    "wap_push_oma_app_internet",
    "wap_push_oma_app_mms",
    "wap_push_oma_app_poc",
    "wap_push_oma_cp_bookmarks",
    "wap_push_oma_settings",
    "wap_push_ota_app_browser",
    "wap_push_ota_app_internet",
    "wap_push_ota_app_mms",
    "wap_push_ota_bookmarks",
    "wap_push_ota_multi_shot",
    "wap_push_ota_settings",
    "wap_push_ota_single_shot",
    "wap_push_ota_support",
    "weight",
    "width",
}
```